src/types.ts

```diff
@@ -74,11 +72,10 @@ export interface SingletonDeploymentV2 {
   released: boolean;
   contractName: string;
   version: string;
-  codeHash: string;
+  deployments: Record<string, Record<string, string>>;
```

Comment:
This type can be more specific
```suggestion
  deployments: Record<AddressType, Record<'address' | 'codeHash', string>>;
```

Reply:
Updated 👍🏾